tutorials/misc/plot_seeg.py

```diff
@@ -0,0 +1,137 @@
+"""
+.. _tut_working_with_seeg:
+
+======================
+Working with SEEG data
+======================
+
+MNE supports working with more than just MEG and EEG data. Here we show some
+of the functions that can be used to facilitate working with
+stereoelectroencephalography (SEEG) data.
+"""
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#          Adam Li <adam2392@gmail.com>
+#
+# License: BSD (3-clause)
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+
+import mne
+from mne.viz import plot_alignment, snapshot_brain_montage
+
+print(__doc__)
+
+# paths to mne datasets - sample SEEG and FreeSurfer subject
+misc_path = mne.datasets.misc.data_path()
+sample_path = mne.datasets.sample.data_path()
+subject = 'sample'
+subjects_dir = sample_path + '/subjects'
```

Comment:
For this example, do not use `sample`. Use `fsaverage`. That way the ECoG example is "ECoG data in a subject's space projected to a surface" and the sEEG example is "sEEG data in MNI space projected to a volume". And people who have ECoG/sEEG x subject/MNI x surf/vol can pick the parts of the tutorials they need. And the narrative docs of the two examples should begin by stating this very clearly, e.g., this should have something like:
```
This example shows how to use:

- sEEG data
- channel locations in MNI space
- projection into a volume

For an example that involves ECoG data, channel locations in a subject-specific MRI, or projection into a surface, see :ref:`tut-working-with-ecog`.
```
And then `plot_ecog.py` can be updated with a similar list+ref.